vm/src/vm/exec_loop.rs

```diff
@@ -948,30 +950,29 @@ impl<ENV> GVm<ENV> {
                 SUB => binary_math!(self, chunk, self.ip_ptr, |a, b| a - b, wide),
                 MUL => binary_math!(self, chunk, self.ip_ptr, |a, b| a * b, wide),
                 DIV => div_math!(self, chunk, self.ip_ptr, wide),
-                NUMEQ => compare_int!(
-                    self,
-                    chunk,
-                    self.ip_ptr,
-                    |a, b| a == b,
-                    |a: f64, b: f64| (a - b).abs() < f64::EPSILON,
-                    wide,
-                    true,
-                    false
-                ),
-                NUMNEQ => compare_int!(
-                    self,
-                    chunk,
-                    self.ip_ptr,
-                    |a, b| a == b,
-                    |a: f64, b: f64| (a - b).abs() < f64::EPSILON,
-                    wide,
-                    true,
-                    true
-                ),
-                NUMLT => compare!(self, chunk, self.ip_ptr, |a, b| a < b, wide, true),
-                NUMLTE => compare!(self, chunk, self.ip_ptr, |a, b| a <= b, wide, true),
-                NUMGT => compare!(self, chunk, self.ip_ptr, |a, b| a > b, wide, true),
-                NUMGTE => compare!(self, chunk, self.ip_ptr, |a, b| a >= b, wide, true),
+                NUMEQ => {
+                    compare_numeric!(self, chunk, self.ip_ptr, |a, b| a == b, wide)
+                }
+                NUMNEQ => {
```

Comment:
Good catch, not= should be (not (=... instead of (not (==... which I made it when I renamed operators.  In the old regime there was a numeric not equal op (/= I think) but now there is not.  Per other discussions, not= could be a macro or we could add a NEQUAL opcode that does this.  This opcode should be restored to numeric, you can leave it for this PR and I can clean this up in a bit.  Do we want a 'not==' for numeric not equal?  Could also be a macro, and maybe these should be, added the negative opcodes when we have NOT may be over-optimizing anyway...